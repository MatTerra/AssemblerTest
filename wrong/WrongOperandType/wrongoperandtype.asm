section data
one: const start;not allowed to use labels
two: const 1.2
three: const 2,0
section text
start:
add 1 ; not allowed to use numbers
output one+1 ; not allowed to sum
output one + 1
stop
