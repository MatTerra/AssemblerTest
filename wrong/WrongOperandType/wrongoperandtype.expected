Line 2 Invalid operand start - Syntax
Line 5 Invalid operand 1 - Syntax OR Line 5 Invalid label 1 - Lexical
